.github/workflows/ci.yml

```diff
@@ -123,8 +123,25 @@ jobs:
     - name: fmt
       run: cargo fmt --all -- --check
 
+  check_docs:
+    timeout-minutes: 30
+    name: Checking docs
+    runs-on: ubuntu-latest
+    env:
+      RUSTC_WRAPPER: "sccache"
+      SCCACHE_GHA_ENABLED: "on"
+    steps:
+    - uses: actions/checkout@v4
+    - uses: dtolnay/rust-toolchain@stable
```

Comment:
If you use `with: toolchain` then this should be `@master`

Reply:
but then this is not against a fixed tag and would change everytime there is a change on the action, no?

Reply:
true, it's the way the action is designed though.  i think it makes sense to use as designed, until we find out it hurts us.  @stable is also a moving tag which moves every 6 weeks.